sonicwall_firewall/assets/logs/sonicwall-firewall.yaml

```diff
@@ -0,0 +1,1090 @@
+      enabled: true
+      source: natSrc
+      samples:
+        - 10.10.10.10:48245
+        - 10.10.10.10
+      grok:
+        supportRules: ""
+        matchRules: natscr_parser_rule %{ip:source.nat.ip}(:%{number:source.nat.port})?
+    - type: grok-parser
+      name: Parsing Sonicwall Firewall nat destination
+      enabled: true
+      source: natDst
+      samples:
+        - 10.10.10.10
+        - 10.10.10.10:320
+      grok:
+        supportRules: ""
+        matchRules: natdst_parser_rule
+          %{ip:destination.nat.ip}(:%{number:destination.nat.port})?
+    - type: grok-parser
+      name: Parsing Sonicwall Firewall protocol and service
+      enabled: true
+      source: proto
+      samples:
+        - tcp/http
+        - icmp
+        - ipv6-icmp
+        - udp/netbios-ns
+      grok:
+        supportRules: protocol_service_parser %{regex("[a-zA-Z0-9-]*")}
+        matchRules: proto_parser_rule
+          %{protocol_service_parser:protocol}(/%{protocol_service_parser:service})?
+    - type: grok-parser
+      name: Parsing Sonicwall Firewall time
+      enabled: true
+      source: time
+      samples:
+        - 2024-09-11 06:30:00 UTC
+      grok:
+        supportRules: ""
+        matchRules: time_parser_rule %{date("yyyy-MM-dd HH:mm:ss z"):timestamp}
+    - type: date-remapper
+      name: Define `timestamp` as the official date of the log
+      enabled: true
+      sources:
+        - timestamp
+    - type: grok-parser
+      name: Parsing Sonicwall Firewall message
+      enabled: true
+      source: msg
+      samples:
+        - Assigned IP address 0.0.0.0 to MAC address 00:00:00:00:0C:0E
+        - SYN-Flooding machine %s blacklisted
+        - TCP-Flooding machine %s blacklisted
+        - Possible ARP attack from MAC address %s
+        - RST-Flooding machine %s blacklisted
+      grok:
+        supportRules: ""
+        matchRules: >-
+          message_parser_rule_1 Assigned IP address %{ip:ip_address} to MAC
+          address %{mac:MacAddress}
+
+          message_parser_rule_2 (TCP|SYN|RST|FIN)-Flooding machine %{greedyData:message_value} blacklisted
+
+          message_parser_rule_3 Possible ARP attack from MAC address %{greedyData:message_value}
+
+          message_parser_rule_4 Possible replay attack with this client IP - %{greedyData:message_value}
+    - type: attribute-remapper
+      name: Map `app` to `syslog_appid`
+      enabled: true
+      sources:
+        - app
+      sourceType: attribute
+      target: syslog_appid
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `af_polid` to `app.policy.id`
+      enabled: true
+      sources:
+        - af_polid
+      sourceType: attribute
+      target: app.policy.id
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `af_policy` to `app.policy.name`
```

Comment:
These seem to be custom attributes for the integration as it doesn't map to any [standard attributes](https://docs.datadoghq.com/standard-attributes/?product=log+management&search=app). If that's the case, then we tend to add a prefix for the integration. Something like `sonicwall.policy_name` to namespace them accordingly

Reply:
Same comment for a bunch of remappers below this

Reply:
Hey @vishalshah-dd , we don't have any facets associated with this custom attributes. Do we still need namespacing ?

Reply:
We thought about this a bit more, and we won't need namespacing if it isn't being exposed as a facet so we're good as is

Reply:
Thanks Vishal.